src/plugins/intel_cpu/tests/functional/single_layer_tests/topk.cpp

```diff
@@ -153,7 +161,10 @@ class TopKLayerCPUTest : public testing::WithParamInterface<TopKLayerCPUTestPara
 
             // For int32, deliberately set big numbers which are not accurately representable in fp32
             int start = netPrecision == ElementType::i32 ? pow(2, 30) + 1 : - static_cast<int>(size / 2);
-            std::iota(data.begin(), data.end(), start);
+
+            // For stabling sorting, explicitly set half of the blob values to be repeating zero
+            size_t set_size = sort == SortType::SORT_VALUES && stable ? size / 2 : size;
+            std::iota(data.begin(), data.begin() + set_size, start);
```

Comment:
I meant something like that:
`std::iota(data.begin(), data.begin() + set_size, start);
std::copy(data.begin(), data.begin() + set_size, data.begin() + set_size);`
Otherwise, there will be only zeros at the top.

Reply:
Done. Thanks Nikolay!